lib/kube_deploy_tools/artifact_registry/driver_gcs.rb

```diff
@@ -84,6 +84,22 @@ def download_artifact(input_path, output_dir_path)
       output_path
     end
 
+    def get_latest_build_number(project)
+      out, err, status = Shellrunner.run_call('gsutil', 'ls -l', '#{@bucket}/project/#{project}/build/')
+
+      if !status.success?
+        raise "Failed to list latest builds for project"
+      end
+      # pick out the build numbers from the list
+      build_regex = /[0-9]+\/$/
```

Comment:
You could run this regex with a [capture group](https://ruby-doc.org/core-2.5.0/String.html#method-i-scan) to remove the slash with less code.

```
irb(main):008:0> 'gs://blargh/project/foo/build/525/'.scan(/([0-9]+)\/$/)[0]
=> ["525"]
```

Observe the parens around the [0-9]+, that's a capture!

Reply:
Aha, cool, thanks!